src/main/cli/settings.c

```diff
@@ -1464,6 +1464,9 @@ const clivalue_t valueTable[] = {
 #if defined(USE_VTX_CONTROL) && defined(USE_VTX_COMMON)
     { "vtx_halfduplex",             VAR_UINT8  | MASTER_VALUE | MODE_LOOKUP, .config.lookup = { TABLE_OFF_ON }, PG_VTX_CONFIG, offsetof(vtxConfig_t, halfDuplex) },
 #endif
+#ifdef USE_VTX_TRAMP
```

Comment:
Thinking of this, can we have the `trampPitModeInverted` removed from this pull request please?
The other changes are good and make sense, but adding an option to support broken protocol implementations does not make sense seeing that the most popular brand with this problem (@MATEKSYS) is looking into fixing it in their VTX firmware.

Reply:
Good call @mikeller, I've updated the PR (adding a second commit - let me know if I should squash them, although would be good to keep the cli option as a reference for now). Have backed out the CLI option. So will be working for official IRC users and temporarily broken for mateksys users again :-P.